src/mobot_client/timeouts.py

```diff
@@ -28,53 +28,53 @@ def __init__(self, messenger, payments, schedule=30, idle_timeout=60, cancel_tim
     @staticmethod
     def customer_is_active(customer):
         session = DropSession.objects.filter(customer=customer).values('state').last()
-        if session is not None and session.state in ItemSessionState.active_states():
+        if session is not None and session.state in SessionState.active_states():
```

Comment:
Nit: This can just be `return session is not None and session.state in SessionState.active_states()`

Reply:
Fixed, though this is sort of out of scope for this PR as I'm just doing a find/replace on this.